src/tinkerio.py

```diff
@@ -910,13 +919,13 @@ def molecular_dynamics(self, nsteps, timestep, temperature=None, pressure=None,
         if temperature is not None:
             md_defs["integrator"] = "stochastic"
         else:
-            md_defs["integrator"] = "beeman"
+            md_defs["integrator"] = "respa"
```

Comment:
The default parameters are what's used if the user doesn't specify the option in the key file, so it doesn't need to be the most efficient, it only needs to be fail-safe.

Does the use of respa require any additional options in the key file, i.e. specifying the long and short timestep, or the subdivision of the long timestep to make the short timestep?